jobs/nightly-rpm-builds.yml

```diff
@@ -15,6 +15,15 @@
           predefined-parameters:
             GERRIT_BRANCH=master
 
+            CENTOS_VERSION=8
+
+            CENTOS_ARCH=x86_64
+    - trigger-builds:
+        - project: gluster_build-rpms
+          block: false
+          predefined-parameters:
+            GERRIT_BRANCH=master
+
```

Comment:
Maybe you want to remove EOL versions from the nightly build too? https://www.gluster.org/release-schedule/ lists 5 and 6 as EOL.